python/GangaDirac/Dirac.ini

```diff
@@ -4,6 +4,12 @@ RUNTIME_PATH = GangaDirac
 
 ReleaseNotes = False
 
+[defaults_GridCommand]
+info = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-info
+init = /cvmfs/ganga.cern.ch/dirac_ui/scripts/dirac-proxy-init -g gridpp_user -M
+[DIRAC]
+DiracEnvFile = GangaDirac/cvmfs_DIRAC_env
```

Comment:
I'm happy to do so, so where should we put it?
I don't know if the current envfile was enough I amalgamated the 2 files setting up env vars into 1 file just in-case.
If you give me a location I'll happily change it but we should probably have it by default here :)